test/elpy-insert--pip-button-value-create-test.el

```diff
@@ -12,8 +12,11 @@
     (mletf* ((executable-find
               (command)
               (cond
-                ((equal command "pip") t)
                 ((equal command "easy_install") t)))
+             (call-process
+              (program &optional infile destination display &rest args)
+               (when (and (equal program "python") (equal args '("-m" "pip"))))
```

Comment:
Yes – that check is not doing anything, whether it works or not. The `should` statement should make sure that we use `python -m pip` now, not just `pip`. That's what I added now, too. :-)